templates/author.twig

```diff
@@ -1,35 +1,25 @@
 {% extends "base.twig" %}
 
 {% block content %}
-
-	<div class="clearfix"></div>
-	<div class="skewed-overlay"></div>
-
-	<header class="page-header">
-		<div class="container">
+	<header class="page-header container">
+		{% if ( author.avatar ) %}
+			<figure class="author-image" aria-label="{{ __( 'Picture of ' ~ author.name , 'planet4-master-theme' ) }}">
+				<img itemprop="image" class="author-pic"
+					 src="{{ fn('get_avatar_url', author.id, {'size' : 300}) }}"
+					 alt="{{ author.name }}">
+			</figure>
+		{% endif %}
+		<div class="author-details">
 			<h1 class="page-header-title">{{ author.name }}</h1>
```

Comment:
I think this `h1` tag is too big compared to the designs, at least on mobile

Reply:
I'll check this with Houssam. This `h1` is using the same style as `h1` anywhere else on the site. Not sure if it was the design's intention to use a different `h1` font size only on this page.

Reply:
We'll keep the general `h1` size for now and maybe get back to this later.